distribution/ddf-common/src/main/resources/etc/templates/example.ManagedService.config

```diff
@@ -1,22 +1,21 @@
-service.pid="example.ManagedService"
 property.string="string"
 property.boolean.true=B"true"
 property.boolean.false=B"false"
 property.int=I"10"
 property.long=L"100"
-property.float=F"10.5"
-property.double=D"100.1234"
+property.float=F"0x41280000"
```

Comment:
❓ Should we add comments in those example files to explain what the format of those values are or refer to the documentation?